crates/ruma-events/src/tag.rs

```diff
@@ -1,9 +1,14 @@
 //! Types for the *m.tag* event.
 
-use std::collections::BTreeMap;
+use std::{
+    collections::BTreeMap,
+    error::Error,
+    fmt::{Display, Formatter, Result as FmtResult},
```

Comment:
Sorry for the nit, but could you only import `fmt` and use the items as `fmt::Display`, `fmt::Formatter`, `fmt::Result` for consistency with the rest of the codebase?